### ISSUE TITLE
WISDEM GUI/operation will not launch-Libshiboken does not exist

### ISSUE THREAD

**Issue by Kashcroft57** [WISDEM/WISDEM#350]
## Description
I've just installed WISDEM, the tests mostly work, 7 failed, 1294 passed. when i try to launch the GUI or run the yaml files without the GUI I get the message below and cannot progress.  I'm completely stumped 

import error: .......shiboken2\libshiboken does not exist 
### Steps to reproduce issue
_Please provide a minimum working example (MWE) if possible_

1. …installed as recommended for windows on main wisdem page 
2. …inserted command (wisdem-env) PS C:\Users\xx.xxxx\WISDEM> wisdem

### Current behavior
…Traceback (most recent call last):
  File "C:\Users\xx.xxxx\Anaconda3\envs\wisdem-env\Scripts\wisdem-script.py", line 33, in <module>
    sys.exit(load_entry_point('WISDEM', 'console_scripts', 'wisdem')())
  File "C:\Users\xx.xxxx\Anaconda3\envs\wisdem-env\Scripts\wisdem-script.py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "C:\Users\xx.xxxx\Anaconda3\envs\wisdem-env\lib\importlib\metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "C:\Users\xx.xxxx\Anaconda3\envs\wisdem-env\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\users\xx.xxxx\wisdem\wisdem\main.py", line 6, in <module>
    from wisdem.inputs.gui import run as guirun
  File "c:\users\xx.xxxx\wisdem\wisdem\inputs\gui.py", line 7, in <module>
    from PySide2.QtWidgets import (  # type: ignore
  File "c:\users\xx.xxxx\anaconda3\envs\wisdem-env\lib\site-packages\pyside2-5.15.2-py3.8-win-amd64.egg\PySide2\__init__.py", line 107, in <module>
    _setupQtDirectories()
  File "c:\users\xx.xxxx\anaconda3\envs\wisdem-env\lib\site-packages\pyside2-5.15.2-py3.8-win-amd64.egg\PySide2\__init__.py", line 54, in _setupQtDirectories
    for dir in _additional_dll_directories(pyside_package_dir):
  File "c:\users\xx.xxxx\anaconda3\envs\wisdem-env\lib\site-packages\pyside2-5.15.2-py3.8-win-amd64.egg\PySide2\__init__.py", line 26, in _additional_dll_directories
    raise ImportError(shiboken2 + ' does not exist')
ImportError: c:\users\xx.xxxx\anaconda3\envs\wisdem-env\lib\site-packages\shiboken2\libshiboken does not exist
(wisdem-env) PS C:\Users\xx.xxxx\WISDEM>

### Expected behavior
…

### Code versions
_List versions only if relevant_
- Python 3.8

**Comment by gbarter** [comment]
I traced the problem to a conflict between duplicate pyside2 installations with conda vs pypi on Windows.  Are you comfortable switching to the `develop` branch of WISDEM?  You will at least need to do the following once #351 gets merged and closed out:
```
pip uninstall pyside2 shiboken2
conda install --force-reinstall qt pyside2
python setup.py develop
```
You may need to completely uninstall and reinstall `qt` and `pyside2` in conda, but I'm crossing my fingers that the above is sufficient.  If this is beyond your comfort level, another approach would be to just start over with a new environment name and redo the installation instructions (once the PR is completed).

**Comment by Kashcroft57** [comment]
Hello
Thank you for the speedy response and action.  Unfortunately it still doesn't seem to work for me. 
I tried the suggested code, still no joy.  So I created  wisdem1 env and attempted installation again, following the instructions here https://wisdem.readthedocs.io/en/master/installation.html, which I understand launches developer mode,  and on entering "WISDEM" to launch the GUI i ultimately received the following  "ImportError: c:\users\xxxx\anaconda3\envs\wisdem1-env\lib\site-packages\shiboken2\libshiboken does not exist" . I tried tagging on your additional code and the transcript ended with "Installing PySide2-5.15.2-5.15.2-cp35.cp36.cp37.cp38.cp39-none-win_amd64.whl to c:\users\xxxx\anaconda3\envs\wisdem1-env\lib\site-packages
Installing shiboken2-5.15.2-5.15.2-cp35.cp36.cp37.cp38.cp39-none-win_amd64.whl to c:\users\xxxx\anaconda3\envs\wisdem1-env\lib\site-packages". I entered "Wisdem" and it returned the same result, libshiboken does not exist. I'm stumped.
